tests/unit/test_select.py

```diff
@@ -217,14 +221,14 @@ def test_disable_quote(faux_conn):
         "t",
         sqlalchemy.Column(quoted_name("foo", False), sqlalchemy.Integer),
     )
-    faux_conn.execute(sqlalchemy.select([table]))
+    faux_conn.execute(sqlalchemy.select(table))
     assert faux_conn.test_data["execute"][-1][0] == ("SELECT `t`.foo \nFROM `t`")
 
 
 @sqlalchemy_before_1_4
```

Comment:
Since we bumped the minimum version to 1.4, we should delete this test. Same for other tests where this annotation appears.

Reply:
Pending some conversations, we may do this in a separate PR